jwst/srctype/tests/test_srctype.py

```diff
@@ -10,231 +9,231 @@
 def test_background_target_set():
 
     # An exposure flagged as background target
-    input = datamodels.ImageModel((10, 10))
+    step_input = datamodels.ImageModel((10, 10))
 
-    input.meta.exposure.type = 'NRS_IFU'
-    input.meta.observation.bkgdtarg = True
-    input.meta.dither.primary_type = '2-POINT-NOD'
-    input.meta.target.source_type = 'POINT'
+    step_input.meta.exposure.type = "NRS_IFU"
+    step_input.meta.observation.bkgdtarg = True
+    step_input.meta.dither.primary_type = "2-POINT-NOD"
+    step_input.meta.target.source_type = "POINT"
 
-    output = srctype.set_source_type(input)
+    output = srctype.set_source_type(step_input)
 
-    # Result should be EXTENDED regardless of other input settings
-    assert output.meta.target.source_type == 'EXTENDED'
+    # Result should be EXTENDED regardless of other step_input settings
+    assert output.meta.target.source_type == "EXTENDED"
 
 
 def test_background_target_unset():
 
     # An exposure without BKGDTARG set at all
-    input = datamodels.ImageModel((10, 10))
+    step_input = datamodels.ImageModel((10, 10))
 
-    input.meta.exposure.type = 'NRS_IFU'
-    input.meta.dither.primary_type = '2-POINT-NOD'
-    input.meta.target.source_type = 'EXTENDED'
+    step_input.meta.exposure.type = "NRS_IFU"
+    step_input.meta.dither.primary_type = "2-POINT-NOD"
+    step_input.meta.target.source_type = "EXTENDED"
 
-    output = srctype.set_source_type(input)
+    output = srctype.set_source_type(step_input)
```

Comment:
In these tests I find the variable name `step_input` to be a bit confusing because the step itself is not being called.  Maybe something like `input_model`?